core/controllers/admin_test.py

```diff
@@ -1249,11 +1248,11 @@ def test_update_username_creates_audit_model(self):
         user_id = self.get_user_id_from_email(self.ADMIN_EMAIL)
         csrf_token = self.get_new_csrf_token()
 
-        current_datetime = datetime.datetime.utcnow()
-        with self.mock_datetime_utcnow(current_datetime):
-            # To get the current time in seconds.
-            time_of_query = (datetime.datetime.utcnow() - datetime.datetime(
-                1970, 1, 1)).total_seconds()
+        current_time_in_millisecs = utils.get_current_time_in_millisecs()
+        mock_get_current_time_in_millisecs = lambda: current_time_in_millisecs
+        with self.swap(
+            utils, 'get_current_time_in_millisecs',
+            mock_get_current_time_in_millisecs):
```

Comment:
Can you add a comment explaining why this swap is needed and what exactly happens (and why) when it is not present?

Reply:
Done.